File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Dictum SDK exists test

require_once __DIR__ . '/../dictum_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = DictumSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# Dictum SDK exists test

require "minitest/autorun"
require_relative "../Dictum_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = DictumSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
